pkg/document/crdt/rht.go

```diff
@@ -99,24 +98,38 @@ func (rht *RHT) Get(key string) string {
 // Has returns whether the element exists of the given key or not.
 func (rht *RHT) Has(key string) bool {
 	if node, ok := rht.nodeMapByKey[key]; ok {
-		return node != nil && !node.isRemoved()
+		return node != nil && !node.IsRemoved()
 	}
 
 	return false
 }
 
 // Set sets the value of the given key.
 func (rht *RHT) Set(k, v string, executedAt *time.Ticket) {
-	if node, ok := rht.nodeMapByKey[k]; !ok || executedAt.After(node.updatedAt) {
+	if node, ok := rht.nodeMapByKey[k]; !ok || node.updatedAt == nil || executedAt.After(node.updatedAt) {
```

Comment:
```suggestion
	if node, ok := rht.nodeMapByKey[k]; !ok || executedAt.After(node.updatedAt) {
```

I think it would be better to remove `nil` check. `node.updatedAt` should always be set and if it is nil unintentionally, it's good to let the system crash rather than prevent it, because we should fix it.

Reply:
I added code to check if `updatedAt` is `nil` because the test code calls the `Set` function with a `nil` value as `excutedAt` when testing `Marshal` and `ToXML`. Should I modify the test code to avoid passing `nil` values?